internal/venice-common/src/main/java/com/linkedin/venice/serialization/avro/AvroProtocolDefinition.java

```diff
@@ -40,7 +40,7 @@ public enum AvroProtocolDefinition {
   /**
    * Used for the Kafka topics, including the main data topics as well as the admin topic.
    */
-  KAFKA_MESSAGE_ENVELOPE(23, 10, KafkaMessageEnvelope.class),
+  KAFKA_MESSAGE_ENVELOPE(23, 11, KafkaMessageEnvelope.class),
```

Comment:
Hold on, are we ready to bump up the protocol already? Typically we commit just the new protocol schema without revving this until fully ready. This will be non-reversible if it hits prod. Let's plan it properly.

Reply:
I do not want to bump up this initially, but lots of integration tests failed without upgrading 10 to 11 here.